tests/e2e/vm-neonvmd/00-assert.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 90
+commands:
+  - script: |
+      set -eux
+      pod="$(kubectl get neonvm -n "$NAMESPACE" example -o jsonpath='{.status.podName}')"
+      kubectl exec -n "$NAMESPACE" $pod -- ssh guest-vm ps aux | grep neonvmd | [ $(wc -l) -eq 1 ] || echo "neonvmd process is not found in the $pod"
```

Comment:
AFAICT this will always succeed because `grep neonvmd` shows up in `ps aux` and matches itself.

Let's use `ps aux -o comm` to fix that?

Reply:
Changed to the ps aux -o comm.